lib/ansible/plugins/lookup/first_found.py

```diff
@@ -32,27 +32,40 @@
       skip:
         type: boolean
         default: False
-        description: Return an empty list if no file is found, instead of an error.
-"""
+        description:
+          When set with ``with_first_found``, causes the calling task to be skipped when no files are matched.  When set and used as
```

Comment:
with_ anything that returns an empty list has this effect, this is not just first_found

Reply:
OK ... do you want me to reword this or what?

Reply:
you can just remove the first phrase, 'skip' just causes the lookup to return empty list if no file is matched

Reply:
also this is different than errors ignore, while skip=false will produce an error (after all terms are processed) errors=ignore deals with more global issues with the lookup, not just 'non of the terms matched a file'.

Reply:
alright, i think it is still worth calling out that "skip" in template context doesn't skip.  I'll reword things and the examples.